Task06/Task06/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task06
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите текст на русском:");
+            string s = Console.ReadLine();
+            string newS = s.ToUpper();
+            Console.WriteLine("Текст транслитерацией:");
+            Console.WriteLine(convert(newS));
+
+
+            Console.ReadKey();
+        }
+        static string convert(string s)
```

Comment:
А вот меоды в классе лучше разделять пустой строкой
```suggestion
        }
        
        static string convert(string s)
```